### PR TITLE
Added confirm password validation and translated validation messages …

### PR DESCRIPTION
**Pull request by sirasistant**
…to spanish

### CODE REVIEW

File: z-validations/z-validations-library/src/main/java/ua/org/zasadnyy/zvalidations/validations/ConfirmPassword.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import ua.org.zasadnyy.zvalidations.ValidationResult;
+
+/**
+ * Created by assistant on 17/2/16.
+ */
+public class ConfirmPassword extends BaseValidation {
+
+    private final Field passwordField;
+
+    public static ConfirmPassword build(Context ctx,Field passwordField){
```
Comment (zasadnyy) [inline z-validations/z-validations-library/src/main/java/ua/org/zasadnyy/zvalidations/validations/ConfirmPassword.java]: Preserve consistent formatting, add space after column

File: z-validations/z-validations-library/src/main/java/ua/org/zasadnyy/zvalidations/validations/ConfirmPassword.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package ua.org.zasadnyy.zvalidations.validations;
```
Comment (zasadnyy) [inline z-validations/z-validations-library/src/main/java/ua/org/zasadnyy/zvalidations/validations/ConfirmPassword.java]: Missing licence information. Should be MIT with your name

**Comment by zasadnyy** [comment]
Thank you for the contribution @sirasistant . To get PR merged please update the formatting and add MIT licence to the new file.

**Comment by sirasistant** [comment]
Done!

**Comment by zasadnyy** [comment]
Thanks a lot @sirasistant , have a great evening.

**Pull request merged**